zoomage_report.CURATED.tsv

```diff
@@ -56187,3 +56187,215 @@ strain	mixed C57BL/6J and SV129	http://www.ebi.ac.uk/efo/EFO_0000601	E-TABM-426
+sampling site	premotor cortex	http://purl.obolibrary.org/obo/UBERON_0016634	E-HCAD-35	SRR9123044-GGCCGATGTAGCCTCG	premotor cortex	mismatch
+sampling site	supraglottic part of larynx	http://purl.obolibrary.org/obo/UBERON_0036263	E-GEOD-139324	SRR10340970-ACGCCAGGTACATGTC	supraglottic part of larynx	mismatch
+sampling site	tip	http://purl.obolibrary.org/obo/UBERON_20018405	E-CURD-12	SAMN11526602-TAAGCGTCAAGAGTCG	tip	mismatch
+stimulus	bile	http://purl.obolibrary.org/obo/UBERON_0001970	E-MTAB-8495	SAMEA8235362-TATTACCGTTCGTCTC	bile	mismatch
+infect	Leptopilina boulardi	http://purl.obolibrary.org/obo/NCBITaxon_63433	E-CURD-56	SRR11263266-TCTGGAGCATGC	Leptopilina boulardi	
+infect	Leptopilina boulardi	http://purl.obolibrary.org/obo/NCBITaxon_63433	E-GEOD-141273	SRR10574226-TATGCGTCATGC	Leptopilina boulardi	
+infect	Leptopilina boulardi	http://purl.obolibrary.org/obo/NCBITaxon_63433	E-MTAB-8698	ERR3833766-GAGTCCGGTGAGGGTT	Leptopilina boulardi	
+inferred cell type - authors labels	epicardium	http://purl.obolibrary.org/obo/UBERON_0002348	E-MTAB-8810	ERR3971664-GTGTTAGGTCAAAGCG	epicardium	
+inferred cell type - authors labels	ionocyte	http://purl.obolibrary.org/obo/CL_0005006	E-CURD-52	SRR11589515-AAGATCCGTTGT	ionocyte	
+inferred cell type - authors labels	synergid	http://purl.obolibrary.org/obo/PO_0000191	E-ENAD-50	SRR7812265	synergid	
+inferred cell type - authors labels	Tregs	http://purl.obolibrary.org/obo/CL_0000815	E-CURD-46	SAMN12356579-CGGACACGTGCTAGCC	regulatory T cell	
+inferred cell type - authors labels	trichoblast	http://purl.obolibrary.org/obo/PO_0000262	E-GEOD-141730	SAMN13520734-CAGCAGCTCACTCTTA	trichoblast	
+inferred cell type - ontology labels	epicardium	http://purl.obolibrary.org/obo/UBERON_0002348	E-MTAB-8810	ERR3971667-GTACTTTGTAAGTGTA	epicardium	
+inferred cell type - ontology labels	ionocyte	http://purl.obolibrary.org/obo/CL_0005006	E-CURD-52	SRR11589514-TATCATAGCGAG	ionocyte	
+inferred cell type - ontology labels	lamellocyte	http://purl.obolibrary.org/obo/CL_0000396	E-CURD-56	SRR11263267-GGGAATTCGGAC	lamellocyte	
+inferred cell type - ontology labels	lamellocyte	http://purl.obolibrary.org/obo/CL_0000396	E-GEOD-141273	SRR10574222-CTGAGTCAGCCA	lamellocyte	
+inferred cell type - ontology labels	lamellocyte	http://purl.obolibrary.org/obo/CL_0000396	E-MTAB-8698	ERR3833766-TCACGAATCCTTGCCA	lamellocyte	
+inferred cell type - ontology labels	preosteoblast	http://purl.obolibrary.org/obo/CL_0007010	E-CURD-12	SAMN11526604-ACAGCTAAGACTGGGT	preosteoblast	
+inferred cell type - ontology labels	trichoblast	http://purl.obolibrary.org/obo/PO_0000262	E-GEOD-141730	SAMN13520735-CGGACTGAGATCCTGT	trichoblast	
+facs marker	CD141+	http://purl.obolibrary.org/obo/CL_0002394	E-GEOD-89232	SAMN05946161	CD141-positive myeloid dendritic cell	mismatch
+facs marker	CD1c+	http://purl.obolibrary.org/obo/CL_0002399	E-GEOD-89232	SAMN05946938	CD1c-positive myeloid dendritic cell	mismatch
```

Comment:
Again, I think I would leave out any of these rather unspecific surface marker annotations. They could potentially map to other cell types too. Same for the "CD138 positive" immunophenotype mapping below.